### DISCUSSION TITLE
Can you tell me which BerkeleyDB JE version you are using in opendj 4.4.9?

### DISCUSSION THREAD

**Discussion by ameinsser** [OpenIdentityPlatform/OpenDJ#388] [Q&A] [answered]
I want to know the BerkeleyDB JE version used in opendj 4.4.9.
Is the JE 18.3.12 version shown in the log correct?
I would appreciate it if you could also tell me how to check the BerkeleyDB JE version.
Please help me.

**Comment by vharseko** [comment]
https://github.com/OpenIdentityPlatform/OpenDJ/blob/0cf4a67b0e8eeb8065d5df84238ee690906131eb/opendj-server-legacy/pom.xml#L225

**Reply by ameinsser** [reply, marked as answer]
thank you for your reply
